TeamSnapSDK/SDK/DataTypes/Leagues/TSDKLeagueCustomDatum.m

```diff
@@ -23,6 +24,21 @@ + (NSString *)SDKREL {
     return @"league_custom_data";
 }
 
+- (instancetype)initWithField:(TSDKLeagueCustomField *_Nonnull)field memberId:(NSString *_Nonnull)memberId teamId:(NSString *_Nonnull)teamId {
+    self = [super init];
+    if(self) {
+        self.leagueCustomFieldId = [field objectIdentifier];
```

Comment:
Do we also need `linkMember`, `linkLeagueCustomField`, and `linkTeam`?

Reply:
I hesitate to do that for a couple reasons:

1) The links sometimes have params specific to the associated object. I'm not sure that we want to associate them.
2) These are convenience initializers for initial Datum creation. We still have to round trip to the server to get actual objects. (see the .app PR that's associated)